app/parsers/stackdriver.py

```diff
@@ -275,12 +261,23 @@ def add_resource():
             add_resource()
 
         elif res_type == "gce_instance":
-            # gce instance return us result images whitch doesn't contains the source_image part.
-            # so we check source_image through the disk resource
+
+            resource_data = {
+                'resource_type': 'compute.googleapis.com/Instance',
+                'name': prop("protoPayload.resourceName").split('/')[-1],
+                'location': prop("resource.labels.zone"),
+                'project_id': prop("resource.labels.project_id"),
+            }
+
+            # Logs are sent for appengine instances, but the google api hides them and will return a 404
+            if not resource_data['name'].startswith('aef-'):
```

Comment:
we also discovered `aet-` instances related to exposing VPC services for AppEngine - do we need to add those here as well?

Reply:
Sounds reasonable to me. Updated